compiler/daml-extension/package.json

```diff
@@ -62,6 +62,10 @@
                 "command": "daml.openDamlDocs",
                 "title": "[DAML Documentation]"
             },
+            {
+                "command": "daml.visual",
+                "title": "Generate visual for daml project"
```

Comment:
Can we come up with a clearer description here? Something like “Visualize the templates and their interactions in the current project”. Maybe we can also point to some longer explanation of how the graphs need to be interpreted? (Not sure if we already have that in the docs but if we don’t we probably should add it).

Reply:
The issue with having that is, it is going to appear in error box if an error occurs and also show up when a user presses ctrl + p